README.md

```diff
@@ -1,158 +1,274 @@
+  <p>The Ewon Ignition Demo was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please visit the Ewon Developer website at <a href="https://developer.ewon.biz/" class="uri">https://developer.ewon.biz/</a> or contact us at <a href="mailto:us-services@hms.se">us-services@hms.se</a>.</p>
+
+  <!-- INFORMATION/SYSTEM DESIGN -->
+  <h3 id="system-design"><span style="font-variant: small-caps;">System Design</span></h3>
+  <p>The Ewon Ignition Demo is a two-part system consisting of an Ewon application and an Ignition Designer project.</p>
+  <p>The Ewon-side application uses FTP to configure tags on the Ewon, then creates simulated tag data and applies it to the Ewon’s tags using the Ewon Java library.</p>
+  <p>The Ignition Designer project, in conjunction with the Ewon Ignition Connector, accesses the data created by the demo’s Ewon application, and displays it in realistic dashboards that are designed to mimic those of a real system.</p>
+
+  <h3 id="trademark-information"><span style="font-variant: small-caps;">Trademark Information</span></h3>
+  <p><small>Ewon&reg; is a registered trademark of HMS Industrial Networks SA. Microsoft&reg; and Windows&reg; are registered trademarks of Microsoft Corporation in the United States and/or other countries. Inductive Automation&reg; is a registered trademark of Inductive Automation, LLC. All other trademarks mentioned in this document are the property of their respective holders.</small></p>
+
+  <!-- GETTING STARTED -->
+  <h2 id="getting-started"><span style="font-variant: small-caps;">Getting Started</span></h2>
+
+  <!-- GETTING STARTED/PREREQUISITES -->
+  <h3 id="prerequisites"><span style="font-variant: small-caps;">Prerequisites</span></h3>
+  <p>These are the items that are required for the installation and use of the Ewon Ignition Demo.</p>
+  <ul>
+    <li>
+      <p><strong>Configured Ewon Flexy Device</strong></p>
+      <ul>
+        <li>
+          <p>For information on how to setup and configure your Ewon device, please refer to <em><a href="#setting-up-an-ewon">Appendix 6.1, Setting Up an Ewon</a>.</em></p>
+        </li>
+      </ul>
+    </li>
+    <li>
+      <p><strong>PC/Computer</strong></p>
+    </li>
+    <li>
+      <p><strong>Ewon Developer ID</strong></p>
+      <ul>
+        <li>
+          <p>For information on how to request and obtain an Ewon Developer ID, please refer to <a href="#obtaining-a-talk2m-developer-id"><em>Appendix 6.2, Obtaining a Talk2M Developer ID</em>.</a></p>
+        </li>
+      </ul>
+    </li>
+    <li>
+      <p><strong>Ewon Ignition Demo Resource Pack</strong></p>
+      <ul>
+        <li>
+          <p>Download latest from <a href="https://github.com/hms-networks/FlexyIgnitionDemo/releases" class="uri">https://github.com/hms-networks/FlexyIgnitionDemo/releases</a> and extract.</p>
+        </li>
```

Comment:
Maybe change to "Download the latest release here:"
 or something

Reply:
changed

Reply:
It would read better 

<p>Download and extract the latest release from <a href="https://github.com/hms-networks/FlexyIgnitionDemo/releases" class="uri">https://github.com/hms-networks/FlexyIgnitionDemo/releases</a>.</p>